java/core/src/test/java/co/worklytics/psoxy/rules/generics/CalendarTest.java

```diff
@@ -16,56 +17,23 @@
 
 class CalendarTest {
 
-    Pattern pattern = Pattern.compile(
-        Calendar.toCaseInsensitiveMultiPattern(Calendar.FOCUS_TIME_BLOCK_SNIPPETS));
-
     RESTApiSanitizerImpl restApiSanitizer = new RESTApiSanitizerImpl(null, null);
 
-
-    @ValueSource(strings = {
-        "Focus Time",
-        "Focus Time Block",
-        "No Meeting Block",
-        "Focus time",
-        "Focus time block",
-        "No meeting block",
-    })
-    @ParameterizedTest
-    public void focusTimeBlockTitleSnippetsExact(String input) {
-        Matcher matcher = pattern.matcher(input);
-
-        assertTrue(matcher.matches());
-
-        String match = matcher.group();
-
-        assertEquals(input, match);
-    }
-
-    @ValueSource(strings = {
-        "Focus Talk Time",
-        "Focus About Blocks",
-        "No Meetings Meeting",
-    })
-    @ParameterizedTest
-    public void focusTimeBlockTitleSnippets_noMatch(String input) {
-        assertFalse(pattern.matcher(input).matches());
-    }
-
     @CsvSource(value = {
         "OOO,OOO",
-        "OOO: Vacation,OOO: ",
-        "OOO Conference,OOO ",
+        "OOO: Vacation,OOO",
+        "OOO Conference,OOO",
         "Out of Office,Out of Office",
-        "Out of the Office: Vacation,Out of the Office: ",
-        "Focus Time,Focus Time",
-        "Secret Project Focus Time,Focus Time",
-        "Focus Time Block,Focus Time Block",
-        "Focus: Secret Project,Focus: ",
-        "No Meeting Wednesday,No Meeting ",
-        " No Meetings,No Meetings",
-        "Prep Time,Prep Time",
-        "Prep Customer Meeting,Prep ",
-        "Prep: Customer,Prep: ",
+        "Out of the Office: Vacation,Out of the Office",
+        "Focus Time,'Focus Time,Focus'",
+        "Secret Project Focus Time,'Focus Time,Focus'",
+        "Focus Time Block,'Focus Time,Focus'",
+        "Focus: Secret Project,Focus",
+        "No Meeting Wednesday,No Meeting",
+        " No Meetings,'No Meetings,No Meetings'", // q: why????
```

Comment:
Because matches with `No Meeting` and `No Meetings`

Reply:
🤔  it shouldn't though ... `No Meeting` token string shouldn't match `No Meetings`, bc I'm transforming it into a pattern with `\b` tokens on either side to only match "word boundaries. So it shouldn't match prefixes.